src/main/java/frc/robot/RobotContainer.java

```diff
@@ -53,15 +63,39 @@ public void configureButtonBindings(){
             true
     ));
 
+    indexerSubsystem.setDefaultCommand(new BaseCommand(indexerSubsystem));
+
+    // Primary controller
     new JoystickButton(primaryController, XboxController.Button.kY.value).whileTrue(
             new RunCommand(() -> {
               swerveSubsystem.zeroGyro();
             })
     );
 
+    // Secondary controller
+    new JoystickButton(secondaryController, XboxController.Axis.kRightTrigger.value).whileTrue(
```

Comment:
How do you keep the shooter spun up?

Reply:
We dont, at least that was the idea. Im sure there is a way to make it a toggle or smth